common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -1347,19 +1347,25 @@ def test_lms_language_change(self):
             u'English'
         )
 
-        select_option_by_text(language_selector, 'Dummy Language (Esperanto)')
+        select_option_by_text(language_selector, 'Dummy Language (Esperanto)', focus_out=False)
```

Comment:
Note that I just saw your response above about these tests-- I put a note above. I am still not understanding why focusing out causes a problem, but we can discuss tomorrow.